addons/hr/res_config.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class HrConfigSettings(models.TransientModel):
+    _name = 'hr.config.settings'
+    _inherit = 'res.config.settings'
+
+    module_hr_gamification = fields.Boolean('Evaluate and motivate the employees', help='The Gamification module provides ways to evaluate and motivate the employees')
```

Comment:
Change help string into : This app gives you tools to challenge employees to reach specific targets. Goals are assigned through challenges to evaluate and compare team members with each other and through time.